src/coord/src/coord.rs

```diff
@@ -3340,6 +3408,35 @@ impl Coordinator {
             }
         };
 
+        // Ensure selection targets are valid, i.e. user-defined tables, or
+        // objects local to the dataflow.
+
+        // 1. Find the ID of all `Get`s, i.e. items read from in the selection.
+        let mut get_ids = Vec::new();
```

Comment:
`selection.global_uses()` does this for you.

Reply:
Ah, ty TIL